src/test/Figure/Population.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class PopulationResponse(CamelCaseModel):
+    label: str = Field(..., title="Data reference label")
+    source: str = Field(..., title="Data source name")
+    value: int = Field(..., title="The population value")
+    updated: str = Field(..., title="Data updated at timestamp text")
```

Comment:
Would be nice to have this field follow some rules for date-times, possibly using a datetime type or such.